README.md

```diff
@@ -96,19 +104,15 @@ await VoxeetSDKExt.breakout.initialize();
 // Create a new room and move participants into it
 const room = {
     name: 'Video games',
-    participants: [
-        participantA,
-        participantB,
-        participantC,
-    ]
-}
+    participants: [participantA, participantB, participantC],
+};
 const roomId = await VoxeetSDKExt.breakout.createRoom(room);
 
 // Move a participant into that previously created room
-await VoxeetSDKExt.breakout.moveTo(roomId, [ participantD ]);
+await VoxeetSDKExt.breakout.moveTo(roomId, [participantD]);
 
 // You can also move a participant back into the main room
-await VoxeetSDKExt.breakout.moveTo(null, [ participantA ]);
+await VoxeetSDKExt.breakout.moveTo(null, [participantA]);
```

Comment:
Could we add proper function to delete/remove a participant from a room ? 
Internally, it will do the same moveTo(null, ...).

Reply:
Added `moveToMainRoom`